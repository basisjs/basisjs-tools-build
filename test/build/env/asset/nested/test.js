importScripts('./test-content.js', './test-content.js');

asset('./test-content.js');
basis.asset('./test-content.js');

asset('./test-content.js', true);
basis.asset('./test-content.js', true);

asset('./test-non-output.js', true);
basis.asset('./test-non-output.js', true);
