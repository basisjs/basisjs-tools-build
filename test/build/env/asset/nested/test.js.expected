  importScripts("res/2UXU8U3kxAevHpp6qHrcjg.js", "res/2UXU8U3kxAevHpp6qHrcjg.js");
  "res/2UXU8U3kxAevHpp6qHrcjg.js";
  "res/sCYCBPgDT7PfyU6bFmdPuQ.js";
  "nested.content // \\ ' \" content\n";
  "content // \\ ' \" content\n";
  "nested.non.output.content\n";
  "non.output.content\n";
